src/lib/components/TestResult.svelte

```
@@ -68,69 +71,75 @@
-						<Table.Cell class="border-r">{$t('Total marks obtained')}</Table.Cell>
-						<Table.Cell>{resultData.marks_obtained} / {resultData.marks_maximum}</Table.Cell>
-					</Table.Row>
+<section class="bg-background flex min-h-screen items-center justify-center px-4 py-10">
+	<div class="w-full max-w-sm overflow-hidden rounded-2xl shadow-sm">
+		<div class="bg-secondary flex flex-col items-center px-6 py-8 text-center">
+			<h2 class="text-foreground mb-1 text-lg font-bold">
+				"{testDetails.name}" {$t('Submitted')}
+			</h2>
+			<p class="text-muted-foreground mb-4 text-sm">
+				{#if testDetails.completion_message}
+					{@html testDetails.completion_message}
```

ESLint

[error] `{`@html`}` can lead to XSS attack.

(svelte/no-at-html-tags)